backends/p4tools/modules/testgen/targets/bmv2/backend/stf/stf.cpp

```diff
@@ -235,7 +235,7 @@ inja::json::array_t STF::getClone(const std::map<cstring, const TestObject *> &c
     return cloneJson;
 }
 
-static std::string getTestCase() {
+std::string STF::getTestCaseTemplate() {
```

Comment:
how are these changes related to unified builds?

Reply:
When multiple `.cpp` files are compiled together this function violates the one-definition rule and the compiler complains.